tasks/add_new_openwrap_partner.py

```diff
@@ -1127,6 +1136,59 @@ def main():
   elif setup_type == constant.ADPOD:
       roadblock_type = 'ONE_OR_MORE'
 
+  if setup_type == constant.ADPOD and deal_lineitem_enabled == True:
+        lib.print_summary(adpod_size, adpod_creative_durations, deal_config, adpod_slots, lineitem_type) 
+        ok = input('Is this correct? (y/n)\n')
+        if ok != 'y':
+            logger.info('Exiting.')
+            return
+        try:
+            # Get the user.
+          user_id = dfp.get_users.get_user_id_by_email(user_email)
```

Comment:
Hi @Pubmatic-Dhruv-Sonone 
There is a significant amount of duplicated code, which may require ongoing maintenance. Is there a way we can avoid this?